middlewares/validators/user.js

```diff
@@ -68,10 +68,11 @@ const validateJoinData = async (req, res, next) => {
       funFact: joi.string().min(100).required(),
       whyRds: joi.string().min(100).required(),
       flowState: joi.string().optional(),
+      numberOfHours: joi.number().min(1).max(100).required(),
```

Comment:
Are we fixing the max to 100?

Reply:
Yes, even 100 is large, as some people have told me 😅

Reply:
Is this weekly or monthly?

Reply:
Weekly.